docs/migration-0.30.1-0.32.x+.md

```diff
@@ -0,0 +1,42 @@
+# Migration from 0.30.1 to 0.32.x+
+
+NOTE: This is not a migration guide from the pre-release versions 0.31.0 and 0.31.1
+
+Version `0.32.0` of this module introduces breaking changes that, without taking additional precautions, will cause the underlying resources to be recreated.
+
+This is because of the newer version's reliance on the [terraform-aws-security-group](https://github.com/cloudposse/terraform-aws-security-group)
+module for managing the module's security group. This changes the Terraform resource address.
+
+To circumvent this, after bumping the module version to the newer version, run a plan to retrieve the resource addresses of
+the SG that Terraform would like to destroy, and the resource address of the SG which Terraform would like to create.
```

Comment:
Feel free to copy from the `elasticache-redis` [migration notes](https://github.com/cloudposse/terraform-aws-elasticache-redis/blob/master/docs/migration-notes-0.41.0.md) to give a more thorough explanation.